### PR TITLE
Add loadgen "stop" mode

### PR DESCRIPTION
**Pull request by marta-lokhova**
Resolves https://github.com/stellar/stellar-core/issues/4488

This should help speed up max TPS runs

### CODE REVIEW

File: src/simulation/test/LoadGeneratorTests.cpp [hunk 1]
```diff
@@ -124,6 +124,26 @@ TEST_CASE("generate load with unique accounts", "[loadgen]")
...
     }
+    SECTION("stop loadgen")
+    {
+        loadGen.generateLoad(GeneratedLoadConfig::createAccountsLoad(
+            /* nAccounts */ 10000,
+            /* txRate */ 1));
+        simulation->crankForAtLeast(std::chrono::seconds(10), false);
+        auto& acc = app.getMetrics().NewMeter({"loadgen", "account", "created"},
+                                              "account");
+        auto numAccounts = acc.count();
```
Comment (SirTyson) [inline src/simulation/test/LoadGeneratorTests.cpp:135]: Nit: Check that completed loadgen runs == 0 here to make sure we didn't just complete the accounts load before running stop.

**Review by SirTyson — APPROVED** [review]
Small test nit but LGTM

File: src/simulation/test/LoadGeneratorTests.cpp (hunk 1 above)
Comment (SirTyson) [reply src/simulation/test/LoadGeneratorTests.cpp:135]: Also is 10 seconds of wall clock a little excessive here?

Comment (marta-lokhova) [reply src/simulation/test/LoadGeneratorTests.cpp:135]: >Nit: Check that completed loadgen runs == 0 here to make sure we didn't just complete the accounts load before running stop.

isn't it what the line below does? it goes from 0->1 on failed runs. 

> Also is 10 seconds of wall clock a little excessive here?
this is loopback mode, so not real time

**Pull request merged**